### ISSUE TITLE
RatesKeeper.Camarero.Handler.init/1 is undefined

### ISSUE THREAD

**Issue by sergibeltran** [am-kantox/camarero#25]
Sometimes happen 3 consecutive times without changing nothing in code and the next one works perfectly.

```
 Compilation error in file lib/rates_keeper/api/outlier.ex ==
** (UndefinedFunctionError) function RatesKeeper.Camarero.Handler.init/1 is undefined (module RatesKeeper.Camarero.Handler is not available)
    RatesKeeper.Camarero.Handler.init([])
    lib/plug/builder.ex:302: Plug.Builder.init_module_plug/4
    lib/plug/builder.ex:286: anonymous fn/5 in Plug.Builder.compile/3
    (elixir) lib/enum.ex:1940: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/plug/builder.ex:284: Plug.Builder.compile/3
    expanding macro: Plug.Builder.__before_compile__/1
    lib/rates_keeper/api/outlier.ex:1: RatesKeeper.Camarero.Endpoint (module)
```

**Comment by am-kantox** [comment]
An attempt to fix made in #26. Since I cannot reproduce the issue, it’s relatively hard to guess whether it was successful or not. Anyway, some valuable cleanup of warnings and `generated: true`.

**Comment by sergibeltran** [comment]
I am still getting this issue on `0.4.1`.

**Comment by am-kantox** [comment]
Now should be a successful attempt: #27 → `v0.4.2`.

**Comment by am-kantox** [comment]
Fixed in d0b934a24ad88403956b26d09943680181175d2d `v0.6.3`.
